docs/userguide/installation.rst

```diff
@@ -1,88 +1,114 @@
 .. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-.. _doc_userguide_install_client:
+.. _doc_userguide_install_parsec:
 
 
-Install Parsec client
-=====================
+Install Parsec
+==============
 
 Windows
 -------
 
-Windows installer is available at the latest stable version on `parsec.cloud <https://parsec.cloud/get-parsec>`_. Otherwise, it is possible to download a specific Parsec version on `GitHub <https://github.com/Scille/parsec/releases/latest>`_, for example to make Parsec work on a 32 bits computer (installers named ``parsec-vX.Y.Z-win32-setup.exe``).
-
+Windows installer is available at the latest stable version on `Get Parsec`_.
 
 Linux
 -----
 
-Parsec is available on Snap:
+AppImage
+^^^^^^^^
+
+AppImage is available at the latest stable version on `Get Parsec`_.
+
+To use Parsec via the AppImage:
+
+1. Download AppImage file
+2. Make it executable
+
+  - Right-click/Properties/Allow executing file as program
+  - Or use the `chmod` command:
+
+    .. code-block:: shell
+
+        chmod u+x Parsec_X.Y.Z_linux_x86_64.AppImage
```

Comment:
We should distribue the appimage in the format `Parsec.AppImage`, that file naming will allow to auto-update the application

Reply:
Should we leave the rest of the sufffix?
```suggestion
        chmod u+x Parsec_linux_x86_64.AppImage
```
Or is it redundant? given our AppImage is always for linux x86_64

Reply:
I left `Parsec.AppImage` for the moment. We can always change it later if needed.